handlers/easy_info/__init__.py

```diff
@@ -0,0 +1,10 @@
+from eventhandler import EventHandler
+
+
+class EasyInfoHandler(EventHandler):
+    def on_issue_labeled(self, api, payload):
+        if str(payload['label']['name']).lower() == 'e-easy':
```

Comment:
Instead, we could just add the `lower` method to `json_cleanup.py`. Explicit type conversions can affect code readability, and it'd be better if we have them as few as possible.